plsql/ExcelTable.pkb

```diff
@@ -5113,6 +5113,7 @@ where t.id = :1
   function getSheets (
     p_file         in blob
   , p_password     in varchar2 default null
+  , p_method       in binary_integer default DOM_READ
```

Comment:
Technically, only large ODS files may require STREAM_READ to get the sheet list, because everything is contained within a single XML document.
For XLSX format, the list comes from the Workbook part, which is always read via XQuery, regardless of the method specified.

Reply:
It's just a comment though, what you did is OK.